### ISSUE TITLE
Verify compiled code works on other platforms.

### ISSUE THREAD

**Issue by xvilo** [oct2pus/emoji-dl#3]
Hi,

Binary doesn't work, could you be able to provide MacOS build? 

```
$ ./emoji-dl 
zsh: exec format error: /[...]/emoji-dl-master/emoji-dl
```

**Comment by oct2pus** [comment]
i should probably mark a commit as a release and post a few binaries for a few platforms.

MacOS might be a bit wonky since I'm cross-compiling (I don't own a Mac) but it should work.

**Comment by oct2pus** [comment]
I have created a [release](https://github.com/oct2pus/emoji-dl/releases/tag/1.0). Please let me know if this works because I do not own a Mac.

**Comment by oct2pus** [comment]
I've been informed that the code works on MacOS (with minor permission annoyances I fixed in release 1.1) and NetBSD. Should probably work on other platforms. 2/5 is fine. Gonna leave this issue open in case anyone has issues.
